rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -4275,9 +4275,22 @@ extern void ppu_precompile(std::vector<std::string>& dir_queue, std::vector<ppu_
 				}
 			}
 
+			if (!src && !Emu.klic.empty() && src.open(path))
+			{
+				src = decrypt_self(src, reinterpret_cast<u8*>(&Emu.klic[0]));
```

Comment:
If the executable is an overlay or full executable, I rather disabling the compilation on process exitspawn due to the long pause it would cause ingame.

But because at this stage it hard to distinguish between PRX and overlay/executable, I suggest doing a file size check. 

So if file size < 0.5 MB, compile it with Emu.klic
If file size >= 0.5 MB, completely discard the result.
In both cases log the debug error.